### PR TITLE
Add documentation for assigning contributors to pull requests to ensure timely merging

### PR DESCRIPTION
**Pull request by janechu**
# Pull Request

## 📖 Description

This pull request adds documentation to the contribution policy which updates the process to include adding a specific assignee. The assignee will act as a final reviewer and it is their responsibility to merge contributions to the repository.

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

**Review by EisenbergEffect — COMMENTED** [review]
I think we should clarify merge responsibility. One of the issues we're having is that a PR will be approved but can still hang out without getting merged. So, maybe it's that the collaborator is responsible for overseeing the PR through to either merge or close and staying on top of that process within reasonable time frames.

File: CONTRIBUTING.md
Comment (chrisdholt) [inline CONTRIBUTING.md:191]: This reads to me that we'll have one person who is "uber" responsible for *review*. I think the issue is less around review and ensuring that once things are fully reviewed they are merged in a timely manner. Can we update this to reflect that as a reality? My concern would be that there is confusion and a scenario arises where the person assigned to the PR is expected to merge once _they_ have approved, rather than once all policies are done and all reviewing comments have been resolved in an appropriate manner.

File: CONTRIBUTING.md
Comment (chrisdholt) [inline CONTRIBUTING.md:199]: Is the expectation that users add folks? Owners are added automatically, but if we are hoping to have collaborator assignment be what manages this we likely need to automate that on our end to ensure that we distribute the work equally and to each responsible party

File: CONTRIBUTING.md
Comment (chrisdholt) [inline CONTRIBUTING.md:203]: Thanks for adding this addendum, I think this will be a good addition. I wonder if for the second sentence we can note that as a _minimum_ - thoughts? I think response time may vary based on the size of the PR, if it relates to existing issues or is just new, etc. I wonder if adding some kind of caveat can help convey that nuance.

File: CONTRIBUTING.md
Comment (chrisdholt) [inline CONTRIBUTING.md:193]: As it relates to the last section, is the expectation that all PR's have been reviewed by all stakeholders _within_ 3 days?

**Comment by janechu** [comment]
@chrisdholt @EisenbergEffect let's talk about the root level issue, that PRs are hanging around too long and not being merged. There are two issues in my mind for what non-automated process could hold up a merge:

1. reviews need to happen
2. someone needs to push the merge button

Right now we are not seeing problem 1 due to the fact that primarily @chrisdholt and @EisenbergEffect take care of reviews, I think this is detrimental for the team in general but it's not holding us up. I believe we should address this situation as well, spreading the weight of the review process around the contributor pool.

I think I need to adjust the wording, but how do we feel about the assignee being the person who is responsible for a PR getting merged. This list of responsibilities includes:

1. Reviewing the code if they are not the creator of the PR
2. Adding any additional reviewers to review if they feel there should be more than what is auto assigned
3. Remind reviewers via discord if the time period starts to stretch before feedback is given that they should review the PR to move it along
4. Merging the code

Thoughts? I think clearly assigning responsibility will be the crux of the solution for moving PRs in.

File: CONTRIBUTING.md
Comment (janechu) [reply CONTRIBUTING.md:193]: I believe the response time should be the same for any feedback that's expected to be submitted. If something is too contentious we could have a different process such as opening a discussion.

File: CONTRIBUTING.md
Comment (janechu) [reply CONTRIBUTING.md:199]: I'm changing this to assignee adding reviewers if they feel it's necessary.

**Comment by janechu** [comment]
I've changed the wording to reflect the above list of responsibilities for the assignee to a pull request.

**Review by EisenbergEffect — APPROVED** [review]

File: CONTRIBUTING.md
Comment (awentzel) [reply CONTRIBUTING.md:206]: Consider dropping the last sentence "This is considered a reasonable period of time" and instead update the title to "Reasonable response time".

Consider adding "The amount of time can vary [ depending on the size of the PR ] ..." to address Chris's concern.

**Review by awentzel — APPROVED** [review]

**Review by chrisdholt — APPROVED** [review]

**Pull request merged**
